### DISCUSSION TITLE
French store issue with "state_id".

### DISCUSSION THREAD

**Discussion by juliovich** [solidusio/solidus#4702] [Troubleshooting]
I'm trying to build a store to sell in France (only in France by the moment).
Now I'm blocked in the checkout process. When I try to test the checkout, I get always the same message:

- Billing address state can't be blank
- Shipping address state can't be blank.

I'm filling the form by hand with a real french address.
![Captura desde 2022-11-05 09-56-16](https://user-images.githubusercontent.com/34285820/200115584-cad67c43-bcd9-40e0-8e0c-cd6fecb1c566.png)

And this is my Zones setting;
![Captura desde 2022-11-05 10-01-52](https://user-images.githubusercontent.com/34285820/200115703-76e53818-b246-416c-887e-1126f57f9a8e.png)

This is my rails server log;
```
Processing by CheckoutsController#update as HTML
  Parameters: {"authenticity_token"=>"[FILTERED]", "order"=>{"email"=>"juliovich@protonmail.ch", "bill_address_attributes"=>{"name"=>"Der Mij", "address1"=>"35, Allée des pins de la ruade ", "address2"=>"", "city"=>"Gujan", "country_id"=>"75", "state_id"=>"", "zipcode"=>"33470", "phone"=>"0787149989"}, "use_billing"=>"1", "ship_address_attributes"=>{"name"=>"Der Mij", "address1"=>"35, Allée des pins de la ruade 33470 Gujan", "address2"=>"", "city"=>"Gujan", "country_id"=>"75", "state_id"=>"", "zipcode"=>"33470", "phone"=>"0787149989"}}, "commit"=>"", "save_user_address"=>"1", "state"=>"address"}
```

**Comment by jarednorman** [comment]
If you search for France in the database (`Spree::Country`), does it have states? I know French addresses don't have a "state" and the subregion can be inferred from the postal code, but there could be a disconnect with how the seed data is set up.

You probably want to update France to not require a state and remove any states associated with it.
